examples/ex02_cksum/test/ChecksumCL_test.py

```diff
@@ -168,6 +169,7 @@ def line_trace( s ):
 #=========================================================================
 # We use source/sink based tests to stress test the checksum unit.
 
+@pytest.mark.usefixtures("cmdline_opts")
```

Comment:
Dumb question: what does this explicit `usefixtures` function call do?

Reply:
this enables a test class to use fixtures